test-clients/src/main/java/com/hedera/services/bdd/suites/contract/integration/EvmSuites.java

```diff
@@ -0,0 +1,136 @@
+package com.hedera.services.bdd.suites.contract.integration;
+
+import com.google.common.base.Splitter;
+import com.google.protobuf.ByteString;
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.infrastructure.meta.ContractResources;
+import com.hedera.services.bdd.suites.HapiApiSuite;
+import com.hedera.services.bdd.suites.contract.openzeppelin.ERC1155ContractInteractions;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.infrastructure.meta.ContractResources.FACTORY_SELF_DESTRUCT_CONSTRUCTOR_CONTRACT;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountInfo;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getContractInfo;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileAppend;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.updateLargeFile;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_SIZE_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_OVERSIZE;
+
+public class EvmSuites extends HapiApiSuite {
+    private final Logger LOGGER = LogManager.getLogger();
+
+    public static void main(String... args) {
+        new EvmSuites().runSuiteAsync();
+    }
+
+    @Override
+    protected List<HapiApiSpec> getSpecsInSuite() {
+        return List.of(
+//                HSCS_EVM_008_SelfDesctructUpdatesHederaAccount(),
+                HSCS_EVM_0014_ContractCreateFailsOnLargeContracts()
+        );
+    }
+
+    // Done
+    private HapiApiSpec HSCS_EVM_008_SelfDesctructUpdatesHederaAccount() {
```

Comment:
I think that it would be better to place the selfdestruct test in the `opcodes` package -> `SelfDestructSuite.java`